### DISCUSSION TITLE
Adminer abandonned, AdminerEvo continuing

### DISCUSSION THREAD

**Discussion by LionelLaffineur** [TimWolla/docker-adminer#161] [General]
Dear @TimWolla,
Since I notice that Adminer was abandoned for some time already and I didn't get any answer to my emails to the original developer proposing my help to continue with him, I forked the project and am trying to continue it.
I already got requests for a docker of the new https://github.com/adminerevo/adminerevo, but I miss both competences with docker and time to learn / maintain the docker.
I was wondering if it would perhaps be interesting that you create and maintain a new docker for AdminerEvo ?
I thank you in advance for your attention.
Regards,
L.

**Comment by TimWolla** [comment]
see also: #147
